en/flight_modes/altitude.md

```diff
@@ -0,0 +1,55 @@
+# ALTITUDE CONTROL Flight Mode
+
+The ALTITUDE CONTROL flight mode maintains constant altitude when all remote control inputs are centered (i.e. no roll, pitch, yaw and at 50% throttle). The mode does not use GPS, and hence will not attempt to hold position against wind.

```

Comment:
For FW it does depend on them being centred - I was trying to make this "generic" but you're quite right that this dependency does not apply to MC and should not be implied. I'll try reword.  Thanks.